pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarRecordCursor.java

```diff
@@ -136,6 +137,10 @@ public PulsarRecordCursor(List<PulsarColumnHandle> columnHandles, PulsarSplit pu
                         pulsarConnectorConfig),
                 new PulsarConnectorMetricsTracker(pulsarConnectorCache.getStatsProvider()));
         this.decoderFactory = decoderFactory;
+        if (pulsarConnectorConfig.getMaxSplitEntryQueueSizeBytes() >= 0) {
```

Comment:
If the size == 0, this means disabled?

Reply:
If the config value is 0, there will be only one entry in the entry queue.